src/lib389/lib389/cli_conf/plugins/automember.py

```diff
@@ -179,10 +182,13 @@ def abort(inst, basedn, log, args):
     if not plugin.status():
         log.error("'%s' is disabled. Abort rebuild membership task can't be executed" % plugin.rdn)
     fixup_task = plugin.abort_fixup(args.DN, args.filter)
-    fixup_task.wait()
+    fixup_task.wait(timeout=args.timeout)
     exitcode = fixup_task.get_exit_code()
     if exitcode != 0:
-        log.error('Abort rebuild membership task for %s has failed. Please, check logs')
+        if exitcode is None:
+            raise ValueError('Abort rebuild membership task has not completed. Please, check logs')
+        else:
+            raise ValueError('Abort rebuild membership task for %s has failed. Please, check logs')
```

Comment:
`task for %s has failed` - %s is a typo as it's not connected to any string. It was in old code but still worth a fix, I think.